### PR TITLE
Sudoku Solver

### LINKED ISSUE #176: Sudoku Solver
I would like to implement the solution to find the sudoku solver in python. 
with proper comments.

please assign me this issue as a part of GSSOC 2021.

### PR DESCRIPTION
**Pull request by Akash20x**
This pull request closes #176

Solution of sudoku solver in python

### CODE REVIEW

File: Code/Python/Sudoku_Solver.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Title: Sudoku Solver
+# Link: https://leetcode.com/problems/sudoku-solver/
```
Comment (plazzy99) [inline Code/Python/Sudoku_Solver.py:2]: we need an appropriate description of the issue.
Links are not supposed to be added.

File: Code/Python/Sudoku_Solver.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+board = [["5","3",".",".","7",".",".",".","."],
```
Comment (plazzy99) [inline Code/Python/Sudoku_Solver.py:4]: not clear

File: Code/Python/Sudoku_Solver.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                    self.columns[j].add(board[i][j])
+                    self.sub_boxes[(i//3,j//3)].add(board[i][j])
+                else:
+                    self.emp[i].append(j)
+        # we work on dict self.emp, pass in the first row number 0, and the first indice value of the dict value list
+        # which includes the column number values.
+        backtrack(0, 0)
+
+solution = Solution()
+solution.solveSudoku(board)
```
Comment (plazzy99) [inline Code/Python/Sudoku_Solver.py:88]: 2 test cases and space and time complexity need to be added here in the bottom

**Review by plazzy99 — CHANGES_REQUESTED** [review]
Update the readme file for the same

**Comment by Akash20x** [comment]
Please check my pull request

**Pull request closed without merging**